### PR TITLE
py-dload: Submission

### PR DESCRIPTION
**Pull request by essandess**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-dload/Portfile [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-dload
+version             0.6
+platforms           darwin
+license             NULL
```
Comment (herbygillot) [inline python/py-dload/Portfile:9]: Please correct the license.

File: python/py-dload/Portfile [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+maintainers         nomaintainer
+
+description         A multipurpose downloader for python > 3.6
+long_description    {*}${description}
+
+homepage            https://github.com/x011/dload
+
+master_sites        https://files.pythonhosted.org/packages/fc/30/41da7ccf08389e9e262dfef55d68acc5e066ab85bfdd26c9039fef57ead9
```
Comment (herbygillot) [inline python/py-dload/Portfile:17]: Is there a reason why we have to specify `master_sites` explicitly? Is the `python` portgroup failing to do something right here?

I see in this Portfile many things manually specified that are usually automatically picked up by the portgroup, such as `livecheck.url` and `extract.suffix`.

**Review by herbygillot — CHANGES_REQUESTED** [review]

File: python/py-dload/Portfile (hunk 1 above)
Comment (essandess) [reply python/py-dload/Portfile]: Thanks done.

File: python/py-dload/Portfile (hunk 2 above)
Comment (essandess) [reply python/py-dload/Portfile:17]: The auto settings don't detect that download hash at PyPi or the whl download name, so both are necessary.

`livecheck` appears to work without these settings, so I deleted these lines.

**Review by herbygillot — APPROVED** [review]

**Pull request merged**